modules/bridge/src/main/ts/ephox/bridge/components/menu/Container.ts

```diff
@@ -0,0 +1,37 @@
+import { FieldSchema, ValueSchema } from '@ephox/boulder';
```

Comment:
This really isn't a menu, it should probably be moved to a subdirectory, eg: `components/menu/card/Container`. The same applies to the other card item sub components.

Reply:
Also maybe this should be called `CardContainer` to avoid confusion?

Reply:
All the Panel sub components are placed directly in `components/dialog/` so to keep it consistent with that, should everything be in `components/menu/` after all?

Reply:
That only ever has 1 top-level component though, this has 4 now so we need to separate it somewhat, otherwise it'll get confusing about what's a sub vs top-level component. Ideally, the dialog stuff should be broken up as well IMO, but that's out of scope.

Reply:
I moved the subcomponents to a `menu/card` subdirectory as I think it makes sense have these in the same place.